README.md

```diff
@@ -31,6 +31,11 @@ Kinto-based systems.
 
 NodeJS v4+ and npm 2.14+ should be installed and available on your machine.
 
+Various pre-commit hooks are available to help prevent you from
+pushing sub-optimal code; to use these, ``pip install --user
```

Comment:
N.B. There are a couple npm packages called `pre-commit` but they appear to be unrelated to the tool by Yelp.